receiver/vcenterreceiver/metadata.yaml

```diff
@@ -296,6 +296,24 @@ metrics:
       value_type: int
       aggregation: cumulative
     attributes: []
+  vcenter.vm.memory.swapped:
+    enabled: true
```

Comment:
Accepting this based on guidelines [here](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/scraping-receivers.md#changing-the-emitted-metrics), which only require new metrics to be disabled by default if the component is beta or stable. This component is still alpha.

Reply:
Thanks for pointing it out, we will keep it in mind in case we'll need to add new metrics in the future,